packages/transaction-controller/src/types.ts

```diff
@@ -74,32 +129,72 @@ export enum WalletDevice {
 }
 
 /**
- * @type Transaction
- *
- * Transaction representation
- * @property chainId - Network ID as per EIP-155
- * @property data - Data to pass with this transaction
- * @property from - Address to send this transaction from
- * @property gas - Gas to send with this transaction
- * @property gasPrice - Price of gas with this transaction
- * @property gasUsed - Gas used in the transaction
- * @property nonce - Unique number to prevent replay attacks
- * @property to - Address to send this transaction to
- * @property value - Value associated with this transaction
+ * Standard data concerning a transaction to be processed by the blockchain.
  */
 export interface Transaction {
+  /**
+   * Network ID as per EIP-155.
+   */
   chainId?: Hex;
+
+  /**
+   * Data to pass with this transaction.
+   */
   data?: string;
+
+  /**
+   * Address to send this transaction from.
+   */
   from: string;
+
+  /**
+   * Gas to send with this transaction.
+   */
   gas?: string;
+
+  /**
+   * Price of gas with this transaction.
+   */
   gasPrice?: string;
+
+  /**
+   * Gas used in the transaction.
+   */
   gasUsed?: string;
+
+  /**
+   * Unique number to prevent replay attacks.
+   */
   nonce?: string;
+
+  /**
+   * Address to send this transaction to.
+   */
   to?: string;
+
+  /**
+   * Value associated with this transaction.
+   */
   value?: string;
+
+  /**
+   * Maximum fee per gas for this transaction.
+   */
   maxFeePerGas?: string;
```

Comment:
We're alphabetical up to here, shall we take the opportunity to order the rest?

Reply:
Done

Reply:
Dunno how did I missed it heh